mesa/mesaPROTON_OC.py

```diff
@@ -1079,8 +1088,32 @@ def choose_intervention_setting(self):
             self.intervention_start = 13
             self.intervention_end = 9999
 
+    def graduate_and_enter_jobmarket(self):
+        """
+        This enables students to move between levels of education and into the labor market.
+        :return: None
+        """
+        primary_age = self.education_levels[1][0]
+        for student in [agent for agent in self.schedule.agents
+                        if agent.education_level == 0 and agent.age() == primary_age and agent.my_school == None]:
+            student.enroll_to_school(1)
+        for school in self.schools:
+            end_age = self.education_levels[school.diploma_level][1]
+            for student in [agent for agent in school.my_students if agent.age() == end_age + 1]:
+                student.leave_school()
+                student.education_level = school.diploma_level
+                if school.diploma_level + 1 in self.education_levels.keys() \
+                        and school.diploma_level + 1 <= student.max_education_level:
+                    student.enroll_to_school(school.diploma_level + 1)
+                else:
+                    student.job_level = extra.pick_from_pair_list(self.work_status_by_edu_lvl[student.education_level][student.gender_is_male], self.rng)
+                    student.wealth_level = extra.pick_from_pair_list(self.wealth_quintile_by_work_status[student.job_level][student.gender_is_male], self.rng)
+                    if student.age() > 14 and student.age() < self.retirement_age and student.job_level == 1 and self.rng.random() < self.labour_status_by_age_and_sex[student.gender_is_male][student.age()]:
```

Comment:
This "14" seems inconsistent with the age of entry into the labor market.

Reply:
Let's adapt this to the new variable.

Reply:
Done! I reported this because it must be changed also in Netlogo. Added to the issue #203